controllers/persistent-volume-claim/controller.go

```diff
@@ -46,10 +44,10 @@ func NewPersistentVolumeClaimReconciler(client client.Client, apiReader client.R
 
 // Reconcile PVC
 func (r *PersistentVolumeClaimReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	logger := log.Log.WithName("persistentvolume-controller").WithValues("Request.Name", req.Name, "Request.Namespace", req.Namespace)
+	logger := log.Log.WithName("persistentvolumeclaim-controller").WithValues("Request.Name", req.Name, "Request.Namespace", req.Namespace)
```

Comment:
I like this name change, we have too much name overlap as it is so clarifying where we can is a good thing